src/button.js

```diff
@@ -4,39 +4,47 @@ import { makeStyles } from '@material-ui/core/styles'
 import MUIButton from '@material-ui/core/Button'
 import clsx from 'clsx'
 
-const useStyles = makeStyles((theme) => {
-  return {
-    label: {
-      color: 'rgba(0, 0, 0, 0.0) !important',
-    },
-    noSpacing: {
-      padding: theme.spacing(0),
-    },
-  }
-})
+const useStyles = color => {
+  const selectStyle = makeStyles((theme) => {
+    const customColor = theme.overrides.MuiButton(color)
+
+    return {
+      containedPrimary: customColor.containedPrimary,
+      outlinedPrimary: customColor.outlinedPrimary,
+      textPrimary: customColor.textPrimary,
+      label: {
+        color: 'rgba(0, 0, 0, 0.0) !important',
+      },
+      noSpacing: {
+        padding: theme.spacing(0),
+      },
+    }
+  })
+
+  return selectStyle()
+}
 
-const Button = ({ children, isLoading, noSpacing, type, ...props }) => {
-  const classes = useStyles()
+const Button = ({ children, isLoading, noSpacing, type, color, ...props }) => {
```

Comment:
Let's add a fallback for `color` so that it doesn't crash when the attribute is mistyped.